src/nightwatch/snapshot.js

```diff
@@ -0,0 +1,37 @@
+const Jimp = require('jimp');
+const fs = require('fs');
+const path = require('path');
+
```

Comment:
This should be written as a custom nightwatch command: http://nightwatchjs.org/guide#writing-custom-commands

This would allow users to set `"custom_commands_path"` and point to a directory in toolkit which contains our custom commands without needing to import this file, `const snapshot = require('../../../src/nightwatch/snapshot');` in every test they need to perform the snapshot.

Reply:
I tried a custom command and printed out the nightwatch API object to identify any possible way to figure it out and there is simply nothing there.

The problem is that nightwatch loads these plugins indirectly so when I find the module parent directory it evaluates to nightwatch and not the test running...